src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,61 @@
+package pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class HomePage extends BasePage{
+
+    public HomePage(WebDriver givenDriver)
+    {
+        super(givenDriver);
+    }
+
+    // Playlist Locators
+    By createPlaylistBtn = By.cssSelector("i[title='Create a new playlist']");
+    By newSimplePlaylistOptionBtn = By.cssSelector("li[data-testid='playlist-context-menu-create-simple']");
+    By newPlaylistInputField = By.cssSelector("input[name='name']");
+    By editPlaylistOptionBtn = By.xpath("//li[contains(@data-testid,'playlist-context-menu-edit')]");
+    By deletePlaylistBtn = By.cssSelector("button[title='Delete this playlist']");
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");